db/schema.rb

```diff
@@ -647,6 +647,7 @@
     t.geography "neighbour_boundary_geojson", limit: {:srid=>4326, :type=>"geometry_collection", :geographic=>true}
     t.string "documents_status", default: "not_started", null: false
     t.datetime "in_committee_at"
+    t.boolean "consultees_checked", default: false, null: false
```

Comment:
I'm going to be annoying... 😬 
Could you use the Review table instead? And then when the reviewer has to check consultees, we can update the review associated with the consultees? 

If that doesn't work, let me know

Reply:
Ahahaha, I think this does make a lot of sense. So treat this step as a sort of "pre-review"? Happy to do this :)